bin/install-wp-tests.sh

```
@@ -0,0 +1,94 @@
+EOF
+
+echo "✓ Wrote ${CONFIG_PATH}"
+
+# Initialise the test database. DROPS the existing DB — only point at a
+# throwaway target.
+if command -v mysql >/dev/null 2>&1; then
+    PWFLAG=""
+    if [ -n "${DB_PASS}" ]; then
+        PWFLAG="-p${DB_PASS}"
+    fi
+    mysql -h"${DB_HOST%:*}" -P"${DB_HOST##*:}" -u"${DB_USER}" ${PWFLAG} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)